howdoi/howdoi.py

```diff
@@ -677,6 +734,10 @@ def command_line_runner():  # pylint: disable=too-many-return-statements,too-man
         _print_ok(__version__)
         return
 
+    if args['sanity_check']:
+        perform_sanity_check()
+        return
```

Comment:
Since we want this to be executed in CI we need to inform CI of failures somehow. The easiest way is to return non-zero in case of failure. See if a `sys.exit()` call with the return value of `perform_sanity_check()` will work here.

Reply:
This should be sorted in the latest commit I pushed, please check it out